rootfs/etc/nginx/template/nginx.tmpl

```diff
@@ -169,6 +170,9 @@ http {
     {{ range $k, $v := $addHeaders }}
     add_header {{ $k }}            "{{ $v }}";
     {{ end }}
+    {{ range $k, $v := $addHeadersAlways }}
+    add_header {{ $k }}            "{{ $v }}" always;
```

Comment:
Because this setting is anyway global, wouldn't it be enough for most users to have a simple flag which adds the `always` option to the `AddHeaders` headers?

Reply:
@antoineco in that case it would not be possible to mix and match though

Reply:
@ElvinEfendi you're right. Continuing the discussion in comments.